docs/pulseload.rst

```diff
@@ -45,6 +34,10 @@ this, you must specify the connection URL in your environment variables or
 
     PULSE_DATA_INGESTION_CONFIG = "amqp://mypulseuserid:mypassword@pulse.mozilla.org:5672/"
 
+or with SSL::
```

Comment:
Perhaps lets just list the SSL option. If people are using pulse.mozilla.org they shouldn't be using it without SSL.